src/main/logins.ts

```diff
@@ -33,8 +34,23 @@ async function migrate(dir: string) {
         'accounts',
         escapeEmailForAccountFolder(account.addr)
       )
-      await fs.move(join(dir, basename(account.path)), join(dir, newFolder))
-      // Backwards compatibility
+      try {
+        await fs.move(join(dir, basename(account.path)), join(dir, newFolder))
+      } catch (error) {
+        log.error(
+          "Moving failed, make sure you don't have multiple account folders for the same account!",
+          error
+        )
+        dialog.showErrorBox(
+          'Account migration',
+          `Migration of ${account.addr} failed,\n` +
+            "make sure you don't have multiple account folders for the same account!\n" +
```

Comment:
make sure you don't have the same e-mail adress multiple times

Reply:
I see your point but your proposed solution misses the point with the account folders

Reply:
`Please make sure that you don't have multiple account folders for the same e-mail address`